docs/source/guides/contributor/chapters/how.rst

```diff
@@ -83,7 +83,7 @@ Git workflow
   manual.
 
 - Make sure your code is working (install your version of avocado, test
-  your change, run ``make check`` to make sure you didn't introduce any
+  your change, run ``python3 setup.py selftests`` to make sure you didn't introduce any
```

Comment:
I think that for now, the `python3 setup.py selftests` is not a substitution for `make check` because the `make check` will restart the environment  (it runs make clean uninstall develop) and then runs selftests.

Reply:
@richtja good point. I'm going to re-work to mention quick the quick tests with `python3 setup.py lint` and then `make check` for the longer tests. 
I have a small annoyance with `make check` because I like more `make selftests` but this is not the PR for discussing that :D